app/main.py

```diff
@@ -2,4 +2,68 @@ class Cargo:
     def __init__(self, weight: int) -> None:
         self.weight = weight
 
-# write your code here
+
+class BaseRobot:
+    def __init__(
+            self,
+            name: str,
+            weight: int | float,
+            coords: list = None
+    ) -> None:
+        self.name = name
+        self.weight = weight
+        self.coords = [0, 0] if coords is None else coords
```

Comment:
You could simplify this expression 
```suggestion
        self.coords = coords if coords else [0, 0]
```
Or use `or` during assignment
```suggestion
        self.coords = coords or [0, 0]
```